source/extensions/stat_sinks/common/statsd/statsd.h

```diff
@@ -88,9 +88,12 @@ class TcpStatsdSink : public Stats::Sink {
   // Stats::Sink
   void flush(Stats::MetricSnapshot& snapshot) override;
   void onHistogramComplete(const Stats::Histogram& histogram, uint64_t value) override {
-    // For statsd histograms are all timers.
-    tls_->getTyped<TlsSink>().onTimespanComplete(histogram.name(),
-                                                 std::chrono::milliseconds(value));
+    // For statsd histograms are all timers, append the ASCII SI symbol to disambiguate.
+    std::string si_name = histogram.unitSymbol().length() > 0
+                              ? fmt::format("{}_{}", histogram.name(), histogram.unitSymbol())
```

Comment:
i'm seeing this pattern in a few places. factor it out to source/common/stats/utility.h.

Reply:
Fixed.